<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(/ui.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui.png);</string>
   </property>
   <widget class="QPushButton" name="t">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>151</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QLabel" name="p">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>311</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_2">
    <property name="geometry">
     <rect>
      <x>447</x>
      <y>151</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_3">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>255</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_4">
    <property name="geometry">
     <rect>
      <x>447</x>
      <y>255</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_5">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>359</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_6">
    <property name="geometry">
     <rect>
      <x>447</x>
      <y>359</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
	background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_7">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>464</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_8">
    <property name="geometry">
     <rect>
      <x>447</x>
      <y>464</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
	background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_9">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>568</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_10">
    <property name="geometry">
     <rect>
      <x>447</x>
      <y>568</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QLabel" name="p_2">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>415</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="p_3">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>520</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="p_4">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>629</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>128</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
	font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>128</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_3">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>231</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>231</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_5">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>336</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>336</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_7">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>440</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_8">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>440</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_9">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>545</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_10">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>545</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_11">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>672</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_12">
    <property name="geometry">
     <rect>
      <x>447</x>
      <y>672</y>
      <width>289</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;
    background-color: #2A5530;
    min-width: 80px;
	color: #FFFFFF;
	font: 22px;
	font-family: Pacifico;
background-image: url(:/track.png);
}


</string>
    </property>
    <property name="text">
     <string>Brain Fluid Explosion Girl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_12">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>653</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_11">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>650</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
    background-color: #9b2100;
    
	color: #FFFFFF;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/ban.png);</string>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fuck">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0px solid;
    border-radius: 15px;

    min-width: 80px;
	color: red;
	font: 25px;
	font-family: Comic Sans Ms;
background-image: url(:/nothinglol.png);
}


</string>
    </property>
    <property name="text">
     <string>fuck go back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="roll">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>170</y>
      <width>59</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
    border-radius: 29px;
  
    
	color: black;
font: 45px;
	font-family: Chasing Embers One;
background-image: url(:/nothinglol.png);</string>
    </property>
    <property name="text">
     <string>Roll!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
